<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="2" failures="0" skipped="0" tests="2" time="1.233" timestamp="2024-01-11T08:02:13.152271" hostname="is247382"><testcase classname="" name="mri.tests.test_fourier_adjoint" time="0.000"><error message="collection failure">ImportError while importing test module '/volatile/Chaithya/Codes/pysap-mri/mri/tests/test_fourier_adjoint.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../Environments/recon/lib/python3.10/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
../../Environments/recon/lib/python3.10/site-packages/_pytest/pathlib.py:567: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
&lt;frozen importlib._bootstrap&gt;:1050: in _gcd_import
    ???
&lt;frozen importlib._bootstrap&gt;:1027: in _find_and_load
    ???
&lt;frozen importlib._bootstrap&gt;:1006: in _find_and_load_unlocked
    ???
&lt;frozen importlib._bootstrap&gt;:688: in _load_unlocked
    ???
../../Environments/recon/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:186: in exec_module
    exec(co, module.__dict__)
mri/tests/test_fourier_adjoint.py:16: in &lt;module&gt;
    from mri.operators import FFT, NonCartesianFFT, Stacked3DNFFT
E   ImportError: cannot import name 'Stacked3DNFFT' from 'mri.operators' (/volatile/Chaithya/Codes/pysap-mri/mri/operators/__init__.py)</error></testcase><testcase classname="" name="mri.tests.test_orc_wrapper" time="0.000"><error message="collection failure">ImportError while importing test module '/volatile/Chaithya/Codes/pysap-mri/mri/tests/test_orc_wrapper.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../Environments/recon/lib/python3.10/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
../../Environments/recon/lib/python3.10/site-packages/_pytest/pathlib.py:567: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
&lt;frozen importlib._bootstrap&gt;:1050: in _gcd_import
    ???
&lt;frozen importlib._bootstrap&gt;:1027: in _find_and_load
    ???
&lt;frozen importlib._bootstrap&gt;:1006: in _find_and_load_unlocked
    ???
&lt;frozen importlib._bootstrap&gt;:688: in _load_unlocked
    ???
../../Environments/recon/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:186: in exec_module
    exec(co, module.__dict__)
mri/tests/test_orc_wrapper.py:16: in &lt;module&gt;
    from mri.operators import (FFT,
E   ImportError: cannot import name 'Stacked3DNFFT' from 'mri.operators' (/volatile/Chaithya/Codes/pysap-mri/mri/operators/__init__.py)</error></testcase></testsuite></testsuites>